client/src/components/Users/form/index.jsx

```diff
@@ -79,12 +90,15 @@ export default class Password extends Component {
           />
           <label htmlFor="confirm" />
           {empty ? (
-            <Error errorMsg={'Can\' be blank'} />
+            <Error errorMsg={"Can' be blank"} />
           ) : passwordError ? (
-            <Error errorMsg={'Passwords don\'t match'} />
+            <Error errorMsg={"Passwords don't match"} />
```

Comment:
@AymanAlqoqa  I think you removed `empty` from state, so the `<Error errorMsg={"Can' be blank"} />` will never be reached.
And also you are using passwordError for both "passwords don't match" and "empty", so maybe line 95 should just use `passwordError` now:
`<Error errorMsg={passwordError} />`